pkgs/development/compilers/rust/1_39_0.nix

```diff
@@ -1,3 +1,8 @@
+# After updating check ...
+# 1. if rustc can produce rust binaries on x86_64-linux, aarch64-linux and x86_64-darwin:
```

Comment:
Don't really agree with this. For example, I have no ability to test on aarch64 or macOS.

Reply:
@GrahamcOfBorg usually can build this in time. For aarch64 we have a beefy community builder: https://github.com/nix-community/aarch64-build-box
which I have access to and usually try to build upgrades.
For macOS alternatively I have a bunch of nixpkgs maintainer that I can ping instead. We followed this pattern for a while until you started merging rust upgrades without testing.